nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/test/java/org/apache/nifi/provenance/ITestPersistentProvenanceRepository.java

```diff
@@ -1307,13 +1186,12 @@ public boolean isAuthorized(ProvenanceEventRecord event, NiFiUser user) {
         assertEquals(7, events.size());
         final List<Long> eventIds = events.stream().map(event -> event.getEventId()).sorted().collect(Collectors.toList());
         for (int i = 0; i < 7; i++) {
-            Assert.assertEquals(i + 3, eventIds.get(i).intValue());
+            assertEquals(i + 3, eventIds.get(i).intValue());
         }
     }
 
-    @Test(timeout = 10000)
+    @Test
```

Comment:
We should probably put an @Timeout here to keep the behavior consistent.

Reply:
Thanks for pointing that out. The test class had a variety of timeouts on different methods. As an integration test, the individual methods completed with a short duration, so I will add a class-level Timeout and revisit other updated tests.